snapcraft/__init__.py

```diff
@@ -139,13 +139,17 @@
   - build-packages: [deb, deb, deb...]
 
     A list of Ubuntu packages to install on the build host before building
-    the part. The files from these packages will not go into the final snap
-    unless they are also explicitly described in stage-packages.
+    the part. The files from these packages typically will not go into the
+    final snap unless they contain libraries that are direct dependencies of
+    binaries within the snap (in which case they'll be discovered via `ldd`),
+    or they are explicitly described in stage-packages.
 
   - stage-packages: [deb, deb, deb...]
 
-    A list of Ubuntu packages must be unpacked in the `stage/` directory.
-    XXX before build? Before stage?
+    A list of Ubuntu packages to be downloaded and unpacked to join the part
```

Comment:
I feel like "to join" is not the right thing to say here.
"unpacked into the part directory" ?